docs/swarm/index.md

```diff
@@ -33,44 +33,45 @@ services. You don't need additional orchestration software to create or manage
+* **Declarative service model:** Docker Engine uses a declarative syntax to let
+you define the desired state of the various services in your application
+stack. For example, you might describe an application comprised of a web front
+end service with message queueing services and a database backend.
 
-* **Desired state reconciliation:** Swarm constantly monitors the cluster state
-and reconciles any differences between the actual state your expressed desired
-state.
+* **Desired state reconciliation:** The Swarm manager node constantly monitors
+the cluster state and reconcile any differences between the actual state your
+expressed desired state.
 
 * **Multi-host networking:** You can specify an overlay network for your
-application. Swarm automatically assigns addresses to the containers on the
-overlay network when it initializes or updates the application.
+services. The Swarm manager automatically assigns addresses to the containers
+on the overlay network when it initializes or updates the application.
 
-* **Service discovery:** Swarm assigns each service a unique DNS name and load
-balances running containers. Each Swarm has an internal DNS server that can
-query every container in the cluster using DNS.
+* **Service discovery:** The Swarm manager assigns each service a unique DNS
+name and load balances running containers. Each Swarm has an internal DNS
+server that can query every container in the cluster using DNS.
 
-* **Load balancing:** Using Swarm, you can expose the ports for services to an
-external load balancer. Internally, Swarm lets you specify how to distribute
+* **Load balancing:** You can expose the ports for services to an
+external load balancer. Internally, the Swarm lets you specify how to distribute
 service containers between nodes.
 
 * **Secure by default:** Each node in the Swarm enforces TLS mutual
 authentication and encryption to secure communications between itself and all
 other nodes. You have the option to use self-signed root certificates or
 certificates from a custom root CA.
 
-* **Scaling:** For each service, you can declare the number of instances you
-want to run. When you scale up or down, Swarm automatically adapts by adding
-or removing instances of the service to maintain the desired state.
+* **Scaling:** For each service, you can declare the number of tasks you want to
+run. When you scale up or down, the Swarm manager automatically adapts by
+adding or removing tasks to maintain the desired state.
 
 * **Rolling updates:** At rollout time you can apply service updates to nodes
-incrementally. Swarm lets you control the delay between service deployment to
-different sets of nodes. If anything goes wrong, you can roll-back an instance
-of a service.
+incrementally. The Swarm manager lets you control the delay between service
+deployment to different sets of nodes. If anything goes wrong, you can
+roll-back an instance of a service.
```

Comment:
I had the same thought, but it's true that you can "roll-back" manually by updating the service back to the old version, so I'm not sure this wording needs any changes.